### ISSUE TITLE
How can I install iYomu on iPad or iPhone?

### ISSUE THREAD

**Issue by FaBRyXoN99** [sendyhalim/iYomu#4]
Hi, how can I install iYomu on iPad or iPhone?

**Comment by karolxcosta** [comment]
Yes i have that issue too. How the heck can i install this on my phone

**Comment by nghianguyen2021** [comment]
I'm trying to build this for iphone but failed. 
I'm using xcode Version 12.2
Anybody help?
Many thanks
